proptest/src/test_runner/reason.rs

```diff
@@ -7,48 +7,201 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use super::backtrace::Backtrace;
 use crate::std_facade::{fmt, Box, Cow, String};
 
 /// The reason for why something, such as a generated value, was rejected.
 ///
-/// Currently this is merely a wrapper around a message, but more properties
-/// may be added in the future.
+/// Contains message which describes reason and optionally backtrace
+/// (depending on several factors like features `backtrace` and
+/// `handle-panics`, and actual spot where reason was created).
 ///
 /// This is constructed via `.into()` on a `String`, `&'static str`, or
 /// `Box<str>`.
-#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Would removal of these traits not be considered public API breaking? It would seem so to me since any client reliant on the methods of the removed traits would no longer work.

(Sorry for getting back to this so late, thank you for your patience 🙇)

Reply:
These had to be implemented manually below - because Backtrace can't implement them, except Debug` which would be just too verbose. Please check lines 83+ and 165+.